### DISCUSSION TITLE
Stuck on the UEFI booting page

### DISCUSSION THREAD

**Discussion by SiegeSailor** [utmapp/UTM#4422] [UTM Help (macOS)] [answered]
I tried to use the Ubuntu Desktop version (*22.04.1* and *20.04.5*), and I was always stuck on the UEFI booting page (Tried to execute every .efl file I could find but all responded with `Command error status: Unsupported`). Is using UTM with an M1 chip to create VM for Ubuntu Desktop impossible so far? If it is, why?

The same thing happens to *Windows 11*.

### Version

- UTM Version: 3.2.4 (58)
- macOS Version: Monterey 12.6
- Mac Chip: M1

### VM Configuration

I have tried **Virtualize** and **Emulate** both. I placed my ISO image in the section `Boot ISO Image:`.

- Storage: 64 GB
- System
    - Hardware
        - Architecture: ARM64 (aarch64)
        - System: QEMU 7.0 ARM Virtual Machine (alias of virt-7.0) (virt)
        - 4096 MB
        - CPU: default
- QEMU
    - Tweaks
        - UEFI Boot: Y
        - RNG Device: Y
        - Use Hypervisor: Y
        - Use local time for base clock: N
        - Force PS/2 controller: N

## Workaround

I followed [this](https://mac.getutm.app/gallery/ubuntu-20-04) to create a VM for Ubuntu Server and then install `ubuntu-desktop`. This should be enough, however, I am using VMs for an OS course and I want to make things consistent with other classmates as much as possible in case of any issues in the future. (Or is there is package list that I should install for turning Ubuntu Server to Ubuntu Desktop?)

**Comment by osy** [comment, marked as answer]
What is the name of the ISO you're using? Make sure you have **arm64** and not **amd64**.

**Reply by SiegeSailor** [reply]
Thank you. I found out that it was **amd64**. The architecture I chose was not compatible with the ISO file.

**Comment by SiegeSailor** [comment]
Although the ubuntu-desktop version can be used on M1, I experienced frequent unknown internal error reports, and the whole experience was extremely laggy. I ended up using the ubuntu-server version and install the desktop package instead.

**Reply by palashkulsh** [reply]
which desktop packages did you use @SiegeSailor for running ubuntu 22 server on utm running m1 host, and how has been your overall experience

**Reply by SiegeSailor** [reply]
`ubuntu-desktop`. It is almost the same as the desktop version, but you have to install some image/audio/game apps by your own.

**Reply by palashkulsh** [reply]
cool thanks siegesailor : )
